doc/Doxyfile.in

```diff
@@ -771,7 +771,7 @@ EXCLUDE_SYMBOLS        =
 # directories that contain example code fragments that are included (see
 # the \include command).
 
-EXAMPLE_PATH           = @CMAKE_CURRENT_SOURCE_DIR@/.. \ # Start by this one to allow documentation of main cmake file
```

Comment:
This will generate a wrong path if Ponca is included as subdirectory in another project